packages/corvid-local-site/src/init.js

```diff
@@ -1,10 +1,31 @@
+const fs = require("fs-extra");
+const path = require("path");
 const reject_ = require("lodash/reject");
+const { UserError } = require("corvid-local-logger");
+
 const initWatcher = require("./watcher");
 const initReadWrite = require("./readWrite");
-const { isPathOfCodeFile, isPathOfDocumentFile } = require("./sitePaths");
+const {
+  isPathOfCodeFile,
+  isPathOfDocumentFile,
+  DEFAULT_FILE_PATHS
+} = require("./sitePaths");
 const { localCodePathToEditorCodePath } = require("./siteConverter");
+const { ERRORS } = require("./messages");
+
+const ensureLocalFileSystemVersion = async siteRootPath => {
```

Comment:
looks like a duplication of the code from: 
packages/corvid-cli/src/commands/open-editor.js